data/core/macros/deprecated-utils.cfg

```diff
@@ -386,3 +386,8 @@ _"No gold carried over to the next scenario."#enddef
     # no interruption of movement, yet prevent the move from being undone.
     {DEPRECATE_ACTION_MACRO "NO_INTERRUPT_NO_UNDO" 3 "1.16" ""}
 #enddef
+
+#define ENABLE_NIGHTBLADE
+    # Place in a campaign or scenario definition to allow Orcish Slayer to advance to Orcish Nightblade.
+    {ENABLE_ADVANCEMENT "Orcish Slayer" "Orcish Nightblade" (set_experience=100)}
```

Comment:
This is wrong. If the advancement is enabled by default, then this macro should just be empty (but with a `#deprecated` statement so there's a warning when someone uses it).

Reply:
Alright, how does it look now? Deprecation level 2 is good?

Reply:
Level 2 is typically fine, yeah. Looks good.

Reply:
Cool, thanks for your help.